platforms/ios/framework/src/TGMarker+Internal.h

```diff
@@ -3,10 +3,11 @@
 //  TangramMap
 //
 //  Created by Karim Naaji on 4/10/17.
+//  Updated by Matt Blair on 7/13/18.
 //  Copyright (c) 2017 Mapzen. All rights reserved.
 //
 
-#import "map.h"
+#include "map.h"
```

Comment:
Why the change here? Import does additional checks to make sure things are only included once.

Reply:
Yeah it's used in other codebases (the mixing) but I don't like it (but again, objc dev being opinionated). Also, the top comment on this answer notes performance improvements by using import: https://stackoverflow.com/a/439691

I don't know if that's still relevant in modern llvm and clang but yeah.

Reply:
Interesting thread, it sounds like some C system headers might rely on `#include` behavior so I'm inclined to do that just in case. Tangram's headers all use `#pragma once` so it should still be easy for the preprocessor.